billy_metadata/or.py

```diff
@@ -0,0 +1,133 @@
+            'display_name': '2007 Regular Session',
+            '_scraped_name': '2007 Regular Session',
+            'slug': '2007R1',
+        },
+        '2008 Special Session': {
+            'display_name': '2008 Special Session',
+            '_scraped_name': '2008 Special Session',
+            'slug': '2008S1',
+        },
+        '2009 Regular Session': {
+            'display_name': '2009 Regular Session',
+            '_scraped_name': '2009 Regular Session',
+            'slug': '2009R1',
+        },
+        '2010 Special Session': {
+            'display_name': '2010 Special Session',
+            '_scraped_name': '2010 Special Session',
+            'slug': '2010S1',
+        },
+        '2011 Regular Session': {
+            'display_name': '2011 Regular Session',
+            '_scraped_name': '2011 Regular Session',
+            'slug': '2011R1',
+        },
+        '2012 Regular Session': {
+            'display_name': '2012 Regular Session',
+            '_scraped_name': '2012 Regular Session',
+            'slug': '2012R1',
+        },
+        '2012 Special Session' : {
+            'display_name': '2012 Speical Session',
+            '_scraped_name': '2012 Special Session',
+            'slug': '2012S1',
+        },
+        '2013 Regular Session': {
+            'display_name': '2013 Regular Session',
+            '_scraped_name': '2013 Regular Session',
+            'slug': '2013R1',
+        },
+        '2013 Special Session': {
+            'display_name': '2013 Special Session',
+            '_scraped_name': '2013 Special Session',
+            'slug': '2013S1',
+        },
+        '2014 Regular Session': {
+            'display_name': '2014 Regular Session',
+            '_scraped_name': '2014 Regular Session',
+            'slug': '2014R1',
+        },
+        '2015 Regular Session': {
+            'display_name': '2015 Regular Session',
+            '_scraped_name': '2015 Regular Session',
+            'slug': '2015R1',
+        },
+        '2016 Regular Session': {
+            'display_name': '2016 Regular Session',
+            '_scraped_name': '2016 Regular Session',
+            'slug': '2016R1',
+        },
+        '2017 Regular Session': {
+            'display_name': '2017 Regular Session',
+            '_scraped_name': '2017 Regular Session',
+            'slug': '2017R1',
+            'start_date': datetime.date(2017, 2, 1),
+            'end_date': datetime.date(2017, 7, 10),
+        },
+    },
+    feature_flags=['influenceexplorer'],
+    _ignored_scraped_sessions=['Today',
+                               '2015-2016 Interim',
+                               '2013 1st Special Session',
+                               '2012 1st Special Session',
+                               '2013 - 2014 Interim',
+                               '2011 - 2012 Interim',
+                               '2009 - 2010 Interim',
+                               '2007 - 2008 Interim']
+)
+
+def session_list():
```

Comment:
Can you move this to a method of the `Oregon` class and rename to `get_session_list`?